index.html

```diff
@@ -10401,8 +10401,11 @@ <h2>Translatable Attributes</h2>
 		<p>To be understandable by assistive technology users, the values of the following <a>states</a> and [=ARIA/properties=] are <a data-cite="html/dom.html#translatable-attributes">translatable attributes</a> and should be translated when a page is localized:</p>
 		<ul>
 			<li><pref>aria-label</pref></li>
+			<li><pref>aria-colindextext</pref></li>
+			<li><pref>aria-description</pref></li>
 			<li><pref>aria-placeholder</pref></li>
 			<li><pref>aria-roledescription</pref></li>
+			<li><pref>aria-rowindextext</pref></li>
 			<li><pref>aria-valuetext</pref></li>
```

Comment:
I'd add the braille properties. For strings of non-braille patterns, it seems useful. I doubt automated translation will try to handle braille patterns, but why not let them try if they want.

Reply:
It's probably fine to add those, because the auto-localizing probably won't work on the Braille unicode range.

However, since we have some author requirements in the braille attributes about using _either braille or non-braille chars, but not both_, there may need to be a user agent note or should req.